### ISSUE TITLE
Deleted note breaks sync process, but it still reports success

### ISSUE THREAD

**Issue by darkk** [nvbn/everpad#373]
I noticed, that everpad could not upload some new notes to the cloud, but remote notes were fetched. Also, everpad reported **successful** sync that was obviously wrong.

I run `everpad-provider --verbose` and got following log:

```
...
Note Geocaching remote
Note 40 000 - lorien local
EDAMNotFoundException(identifier='Note.guid', key='0c3dfb5b-e69a-4731-96f7-e3f02d5b19e2')
```

The note with this GUID was already removed from evernote cloud, so I dropped it from the table using `delete from notes where guid = '0c3dfb5b-e69a-4731-96f7-e3f02d5b19e2'`.

After that fix everpad was able to upload new notes to the cloud.
Version: 2.5.7-0~saucy from PPA

**Comment by TerrificJ** [comment]
yeah, I once have the same problem. run sudo apt-get remove everpad   and sudo apt-get autoremove, then install everpad again. Don't try to sync manually, since I think there is a bug here. If you can fix it by modifying the code, just wait the next update. I think it would work properly without force sync.
